### PR TITLE
FE - Adding Address Input Fields

### LINKED ISSUE #11: FE - Shipping address
As a user, I should be able to edit my shipping address on my profile page.

Tasks:
- [x] User has an input field to enter shipping address on edit profile page.
- [x] On submit, in same post request as image submission, adding email addressas another field being sent in request.
- [x] User redirected to profile page with updated email displayed.

### PR DESCRIPTION
**Pull request by Karlfunhouse**
# PR Description
Please include a summary of the change:
- Adding functionality for a user to update/change their Shipping Address.
- Made sure all properties are updated in actions.js file

Commit Emojis:
:crystal_ball: JS modifications/updates.

If it is a new feature, describe the feature and the iteration of the project spec it addresses.
- Shipped address feature
- closes #11 

Note any questions or struggles as well.
- none for this feature

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
- [x] My code follows the Turing style guidelines
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: code/web/src/modules/user/EditProfile.js [hunk 1]
```diff
@@ -76,7 +79,15 @@ class EditProfile extends Component {
   onChange = (e) => {
     let user = this.state.user
     user[e.target.name] = e.target.value
-    console.log('bananas')
```
Comment (rachael-t) [inline code/web/src/modules/user/EditProfile.js:79]: Thanks for removing this 😂 🍌

File: code/web/src/modules/user/EditProfile.js [hunk 2]
```diff
@@ -76,7 +79,15 @@ class EditProfile extends Component {
...
+
+    this.setState({
+      user
+    })
+  }
+
+  onChange = (e) => {
```
Comment (rachael-t) [inline code/web/src/modules/user/EditProfile.js:88]: Happy we were able to make this dynamic!

**Review by rachael-t — APPROVED** [review]
Looks good! We'll focus on styling the EditProfile page tomorrow!

**Comment by zachholcomb** [comment]
Looks great guys!

**Pull request merged**